documentation/proc-pages/physics-models/fusion_reactions/plasma_reactions.md

```diff
@@ -0,0 +1,236 @@
+# Fusion Reactions
+
+## Overview
+
+The most likely fusion reaction to be utilised in a power plant is the
+deuterium-tritium reaction:
+
+$$
+\mathrm{D + T} \Longrightarrow \mathrm{^{4}He + n + 17.6 \,MeV}
+$$
```

Comment:
Put all the key reactions up here in the same place and just reference them later. Could add ###sections for each reaction then, ie DT, DD, D3He etc